File: LogginColombiaGold/frmChangeLoggin.vb
Imports System.Security.Cryptography
Imports System.Text
Public Class frmChangeLoggin
    Dim oRf As New clsRf()
    Private Sub btnAccept_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnAccept.Click
        Try
            'Valida que la contraseña nueva sea la correcta
            If txtNewPass.Text.ToString() <> txtRepPass.Text.ToString() Then
                MessageBox.Show("Different New Password and Repeat Password", "Logging", MessageBoxButtons.OK, MessageBoxIcon.[Error])
                Return
            End If

            Dim sPwd As String = GetSHA1(txtNewPass.Text.ToString())
            Dim sPwdOld As String = GetSHA1(txtOldPass.Text.ToString())

            Dim sResp As String = oRf.UpdatePass(sPwdOld.ToString(), sPwd.ToString(), clsRf.sUser.ToString())


            MessageBox.Show(sResp)
        Catch ex As Exception
            MessageBox.Show("Error: " + ex.Message)
        End Try
    End Sub
    Public Shared Function GetSHA1(ByVal texto As [String]) As String
        Try
            Dim sha1 As SHA1 = SHA1CryptoServiceProvider.Create()
            Dim textOriginal As [Byte]() = ASCIIEncoding.[Default].GetBytes(texto)
            Dim hash As [Byte]() = sha1.ComputeHash(textOriginal)
            Dim cadena As New StringBuilder()
            For Each i As Byte In hash
                cadena.AppendFormat("{0:x2}", i)
            Next
            Return cadena.ToString()
        Catch ex As Exception
            Throw New Exception(ex.Message)
        End Try
    End Function
End Class

File: LogginColombiaGold/frmChangeLoggin.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using System.Security.Cryptography;

namespace LogginColombiaGold
{
    public partial class frmChangeLoggin : Form
    {
        clsRf oRf = new clsRf();

        public frmChangeLoggin()
        {
            InitializeComponent();
        }

        private void frmChangeLoggin_Load(object sender, EventArgs e)
        {

        }

        public static string GetSHA1(String texto)
        {
            try
            {
                SHA1 sha1 = SHA1CryptoServiceProvider.Create();
                Byte[] textOriginal = ASCIIEncoding.Default.GetBytes(texto);
                Byte[] hash = sha1.ComputeHash(textOriginal);
                StringBuilder cadena = new StringBuilder();
                foreach (byte i in hash)
                {
                    cadena.AppendFormat("{0:x2}", i);
                }
                return cadena.ToString();
            }
            catch (Exception ex)
            {
                throw new Exception(ex.Message);
            }

        }

        private void btnAccept_Click(object sender, EventArgs e)
        {
            try
            {
                //Valida que la contraseña nueva sea la correcta
                if (txtNewPass.Text.ToString() != txtRepPass.Text.ToString())
                {
                    MessageBox.Show("Different New Password and Repeat Password", "Logging", MessageBoxButtons.OK, MessageBoxIcon.Error);
                    return;
                }

                string sPwd = GetSHA1(txtNewPass.Text.ToString());
                string sPwdOld = GetSHA1(txtOldPass.Text.ToString());

                string sResp = oRf.UpdatePass(sPwdOld.ToString(),
                    sPwd.ToString(),
                    clsRf.sUser.ToString());

                MessageBox.Show(sResp);

            }
            catch (Exception ex)
            {
                MessageBox.Show("Error: " + ex.Message);
            }
        }
    }
}
